CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 ## Changelog
 
 ### 2.3.2 (Next)
+* [#45](https://github.com/alexa-js/alexa-app-server/pull/45): Added support for ca chain certificates (#17) - [@tejashah88](https://github.com/tejashah88).
+* [#45](https://github.com/alexa-js/alexa-app-server/pull/45): Added option to specify passphrase for unlocking specified SSL files - [@tejashah88](https://github.com/tejashah88).
+* [#45](https://github.com/alexa-js/alexa-app-server/pull/45): Modified tests to include new SSL handling features - [@tejashah88](https://github.com/tejashah88).
+* [#45](https://github.com/alexa-js/alexa-app-server/pull/45): Upgraded alexa-verifier-middleware to 0.1.8 from 0.1.4 - [@tejashah88](https://github.com/tejashah88).
```

Comment:
I assume this is a required change? That is the server no longer works with the older version? If so this is fine, otherwise it shouldn't be necessary.